src/please.go

```diff
@@ -1005,7 +1040,23 @@ func main() {
 		defer pprof.WriteHeapProfile(f)
 	}
 
-	if !buildFunctions[command]() {
+	return command
+}
+
+
+
+func main() {
+	command := initBuild(os.Args)
+
+	runWatchedBuild = func(watchedProcessName string) {
+		buildFunctions[watchedProcessName]()
+	}
+	success := buildFunctions[command]()
+
+	metrics.Stop()
+	build.StopWorkers()
```

Comment:
can you confirm that these still get stopped even if you `plz run` a target? In that case I don't think it will ever hit this line.

Reply:
Yes! I can definitely confirm that.